app/views/barcode_items/_barcode_item_lookup.html.erb

```diff
@@ -1,2 +1,2 @@
-<% # Increment a counter on how many we've seen so that we can create a unique ID -- really only necessary for test hooks # %>
-<input type="text" id=<%= "_barcode-lookup-#{index}" %> class="__barcode_item_lookup form-control" value="" placeholder="Barcode Entry" data-organization-id="<%= params[:organization_id] %>" />
+<%# Increment a counter on how many we've seen so that we can create a unique ID -- really only necessary for test hooks %>
+<input type="text" id="<%= "_barcode-lookup-#{index}" %>" class="__barcode_item_lookup form-control" value="" placeholder="Barcode Entry" data-organization-id="<%= params[:organization_id] %>" autocomplete="none">
```

Comment:
From what I can [read](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete), the default should be "on", not "none".